tests/run-tests.sh

```diff
@@ -3,68 +3,41 @@
 set -o errexit;
 set -o nounset;
 
-#let the caller supply an ELM_TEST binary if desired
+# Let the caller supply an ELM_TEST binary if desired.
 if [ -z "${ELM_TEST:-}" ]; then
     ELM_TEST=elm-test;
 fi
 
-# since elm/core is treated specially by the compiler (it's always
+# Since elm/core is treated specially by the compiler (it's always
 # inserted as a dependency even when not declared explicitly), we use
 # a bit of a hack to make the tests run against the local source code
 # rather than the elm/core source fetched from package.elm-lang.org.
 
-# create a local directory where the compiler will look for the
+# Create a local directory where the compiler will look for the
 # elm/core source code:
 
 DIR="$(dirname $0)";
 
 cd "$DIR";
 
-export ELM_HOME="$(pwd)/.elm";
-
+export ELM_HOME="$PWD/.elm";
 rm -rf "$ELM_HOME" && mkdir -p "$ELM_HOME";
-
-# elm-test also puts some things in elm-stuff, start with a clean
-# slate there as well
-
 rm -rf elm-stuff;
 
-# now make an initial run of the tests to populate .elm and elm-stuff;
-# this will test against elm/core from package.elm-lang.org, so we
-# don't really care what the results are; we just need to force all
-# the *other* dependencies to be fetched and set up.
-
-echo "seeding framework for test dependencies ...";
-
-# '|| true' lets us ignore failures here and keep the script running.
-# useful when developing a fix for a bug that exists in the version of
-# elm/core hosted on package.elm-lang.org
-"${ELM_TEST}" tests/Main.elm --fuzz=1 > /dev/null || true;
-
-# clear out the copy of elm-core fetched by the above and replace it
-# with the local source code we want to actually test
-
-VERSION_DIR="$(ls ${ELM_HOME}/0.19.1/packages/elm/core/)"
-CORE_PACKAGE_DIR="${ELM_HOME}/0.19.1/packages/elm/core/$VERSION_DIR"
+# Create a link to the git package
+CORE_LINK="${ELM_HOME}/0.19.1/packages/elm/core/1.0.5"
```

Comment:
You might want to add a comment here saying that setting this to `1.0.5` is fine as long as the strategy used by the dependency solver in elm-test (here `elm-json solve`) prioritizes installed versions. In such case, it will use the linked `1.0.5` to the git repo even when a new 1.0.6 version of core exists. This of course only holds true as long as the build-time and run-time tests dependencies do not require a version of `elm/core` > `1.0.5`.

Otherwise, we need to bump this `1.0.5` once there is a new release of `elm/core`.

Reply:
How about this: https://github.com/elm/core/pull/1104/commits/a888bfedfddd7d505808d017e8e6519017c65dfd hopefully this will keep us in sync regardless of the dependency resolution

Reply:
I suppose this does the trick in 99% of the cases. The only exception I can think of is for the commit that changes the package version. It will try to use the new version that does not exist in the package server yet and thus will make the elm compiler fail I think. Might be annoying if this shows a red cross on the commit corresponding to new package push.
What do you think?

Reply:
hmm, this is a problem. Maybe we need to go back to the old approach of running elm-test once to prepopulate elm_home and then simlink

Reply:
Well you could also just ask the package server which version of elm-core is the latest.

Reply:
```sh
curl -L https://package.elm-lang.org/all-packages | jq -r '."elm/core" | .[-1]'
# 1.0.5
```
Maybe there is a better endpoint than `all-packages` but I'm not aware of one.